src/input/SimModSuite.h

```diff
@@ -527,6 +527,31 @@ class SimModSuite : public MeshInput {
         }
       }
     }
+    if (MeshAtt.lFaceIdNoMesh.size() > 0) {
+      for (std::list<int>::iterator it = MeshAtt.lFaceIdNoMesh.begin(); it != MeshAtt.lFaceIdNoMesh.end();
+           it++) {
+        pGEntity face = GM_entityByTag(model, 2, *it);
+        if (face == NULL) {
```

Comment:
NULL should be replaced by nullptr in modern C++

Reply:
(Is more typesafe, as it cannot be implicitly converted to an int)

Reply:
fixed.